src/sentry/utils/data_scrubber.py

```diff
@@ -45,6 +44,7 @@ class SensitiveDataFilter(object):
     """
     MASK = '*' * 8
     VALUES_RE = re.compile(r'\b(?:\d[ -]*?){13,16}\b')
+    URL_PASSWORD_RE = re.compile(r'\b([\'"]?(?:[a-z0-9]+:)?//[^:]+:)([^@]+)@')
```

Comment:
We shouldn't be trying to deal with things like Python reprs (i.e. the regexp should not care that there are quotes). Instead we should either:

1. Find if there are *any* URLs matching a password
2. Scrub the entire value, or just the password part